src/importmeta.d.ts

```diff
@@ -7,22 +7,43 @@ interface ImportMeta {
     CUSTOM_ORGANIZATION_DOMAINS: {
       [key: string]: string
     }
+    EMAILJS_SERVICE_ID: string
+    EMAILJS_TEMPLATE_ID: string
+    EMAILJS_PUBLIC_ID: string
     features: {
       faucet: boolean
       vote: {
         anonymous: boolean
         overwrite: boolean
         secret: boolean
+        customization: boolean
       }
       login: string[]
       census: string[]
+      voting: string[]
+      unimplemented_census: string[]
       languages: string[]
       _census: {
         spreadsheet: boolean
         token: boolean
         web3: boolean
         csp: boolean
       }
+      _unimplemented_census: {
+        phone: boolean
+        email: boolean
+        crm: boolean
+        database: boolean
+        digital_certificate: boolean
+        others: boolean
+      }
+      _voting: {
+        single: boolean
+        multi: boolean
+        approval: boolean
+        participatory: boolean
+        borda: boolean
+      }
```

Comment:
I don't see these in features.ts, and they should be defined there, like it's done with `_census`.